### ISSUE TITLE
Resynthesis of MMV006357

### ISSUE THREAD

**Issue by kym834** [OpenSourceMycetoma/Series-2-Aminothiazoles#4]
Hi @bendndi @drc007 @wwjvdsande,

I've had a look at the synthesis of MMV006357 and would be happy to resynthesis it here at The University of Sydney and ship to @wwjvdsande & co. so that the activity can be confirmed and kick off this series. Is there still an interest in pursuing it?

I was also wondering, in regard to doing some synthesis for a SAR study, would the activity need to be confirmed _in vitro_ and _in vivo_ before we set about synthesising some analogues in which case we would have to wait until the northern hemisphere summer for _in vivo_ testing results? Or, would activity confirmation _in vitro_ be enough that we could start exploring the synthesis of analogues sooner?

**Comment by wwjvdsande** [comment]
Hi @kym834 @bendndi @drc007, MMV006357, MMV675968 and MMV022478 all had potent in vitro and in vivo activity. In the larvae model they performed well. For these compounds the in vivo activity has therefore been proven and in my opinion they would be good starting points for new compound series. If you would willing to make a series out of MMV006357 we could certainly screen in _in vitro_ in the wintertime and than start testing _in vivo_ when the spring comes.

**Comment by bendndi** [comment]
@kym834  I think it could be great to resynthesize a batch of 006357 and perhaps make a handful of close analogues at the same time. If you make some suggestion on a set of analogues I'm happy to chime in and help prioritize  / enhance the list. The easiest way to track would be to have a SMILES list of structure ideas somewhere, but if that's not possible a chemdraw picture would work in the first instance.
I would not worry about the timing of the synthesis relative to the assays, compounds can always just sit in a fridge waiting for the assay to be avaialble.

**Comment by drc007** [comment]
@wwjvdsande @bendndi @kym834 A lot of aminothiazoles are commercially available, whilst it would be very useful to have a fresh supply of MMV006357 available and retested, perhaps we can scope out the SAR with commercial samples?

https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/2

https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/2#issuecomment-370538115

**Comment by MFernflower** [comment]
https://www.frontiersin.org/articles/10.3389/fmicb.2018.02542/full 

@drc007 @bendndi 

Aminothiazoles are thought to work via inhibition of aerobic respiration

this does not discount the possible uses of this compound (E.G azoxystrobin causes ATP levels to plummet but is rather safe)  but animal cell tox might need to be watched out for

**Comment by MFernflower** [comment]
@kym834 @drc007 I agree it would be wise to make a few simplified analogues of this compound -  N,4‐diphenyl‐1,3‐thiazol‐2‐amine to me sounds like an easy one to bang out!

**Comment by MFernflower** [comment]
@wwjvdsande Have any other succinate dehydrogenase inhibitors (besides this one) been tested vs eumycetoma?

**Comment by MFernflower** [comment]
@wwjvdsande @bendndi @Wilson-Lm @mattodd  While we cannot yet confirm the MOA of the aminothiazole as being a succinate dehydrogenase inhibitor or perhaps a glycolysis inhibitor - is there any data of commercial SDHI's (boscalid, carboxin, benodanil or flutolanil) vs Madurella mycetomatis?

**Comment by wwjvdsande** [comment]
@MFernflower not than I'm aware of

**Comment by bendndi** [comment]
Here are some ideas for very close scan analogues around MMV006357:
![image](https://user-images.githubusercontent.com/36627399/93071258-35f95180-f680-11ea-994d-52275a4d3ec6.png)


And same as SMILES:


SMILES | Tag
-- | --
Cc1ccnc(Nc2nc(cs2)c3ccccn3)c1 | MMV006357
Cc1ccnc(Nc2nc(cs2)c3ccccc3)c1 | AT_BP_001
Cc1ccnc(Nc2nc(cs2)c3ccncc3)c1 | AT_BP_002
Cc1ccnc(Nc2nc(cs2)c3cccnc3)c1 | AT_BP_003
N(c1ccccn1)c2nc(cs2)c3ccccn3 | AT_BP_004
Cc1cccc(Nc2nc(cs2)c3ccccn3)c1 | AT_BP_005
Cc1ccnc(Nc2ncc(s2)c3ccccn3)c1 | AT_BP_006
Cc1ccnc(Nc2nc(cs2)c3ccccc3F)c1 | AT_BP_007
Cc1ccnc(Nc2nc(cs2)c3ccc(Cl)cc3)c1 | AT_BP_008
COc1ccc(cc1)c2csc(Nc3cc(C)ccn3)n2 | AT_BP_009
Fc1ccccc1Nc2nc(cs2)c3ccccn3 | AT_BP_010
C(Nc1nc(cs1)c2ccccn2)c3ccccn3 | AT_BP_011
O=C(Nc1nc(cs1)c2ccccn2)c3ccccc3 | AT_BP_012

**Comment by drc007** [comment]
As might be expected most of these aminothiazoles are known compounds, I've added the InChiKey and searched a variety of online databases

SMILES | Tag | InChIKey | ChEMBL | ZINC | eMolecules | Mcule | MolPort
--|--|--|--|--|--|--|--|
Cc1ccnc(Nc2nc(cs2)c3ccccn3)c1 | MMV006357 | QMEBVRXZLKAAIG-UHFFFAOYSA-N | CHEMBL476339 | ZINC000000088330 | 1509070 | MCULE-5313198933 | MolPort-002-013-562
Cc1ccnc(Nc2nc(cs2)c3ccccc3)c1 | AT_BP_001 | YIJMAUFKBCAMPF-UHFFFAOYSA-N | CHEMBL1622956 | ZINC000000190240 | 1321658 | MCULE-3588516968 | MolPort-001-958-994
Cc1ccnc(Nc2nc(cs2)c3ccncc3)c1 | AT_BP_002 | QONSPCMVAHVLGB-UHFFFAOYSA-N | CHEMBL3321862 | ZINC000000088501 | 1509148 | MCULE-9214113868 | MolPort-000-690-546
Cc1ccnc(Nc2nc(cs2)c3cccnc3)c1 | AT_BP_003 | JPOFCGIZOBPUSQ-UHFFFAOYSA-N |  | ZINC000000088711 | 1509104 | MCULE-4995452320 | MolPort-000-690-559
N(c1ccccn1)c2nc(cs2)c3ccccn3 | AT_BP_004 | RYCUBTFYRLAMFA-UHFFFAOYSA-N | CHEMBL489770 | ZINC000000257247 | 1284347 |  | MolPort-000-006-123
Cc1cccc(Nc2nc(cs2)c3ccccn3)c1 | AT_BP_005 | DQQQXXNDHIYTLC-UHFFFAOYSA-N | CHEMBL567008 | ZINC000000088164 | 1260877 | MCULE-7011722160 | MolPort-000-690-520
Cc1ccnc(Nc2ncc(s2)c3ccccn3)c1 | AT_BP_006 | GAAZWVOHHYHPRR-UHFFFAOYSA-N |  |  |  |  | 
Cc1ccnc(Nc2nc(cs2)c3ccccc3F)c1 | AT_BP_007 | CXMUWUHCBOUUOJ-UHFFFAOYSA-N |  |  |  |  | 
Cc1ccnc(Nc2nc(cs2)c3ccc(Cl)cc3)c1 | AT_BP_008 | SBWIYNYNJAKRHN-UHFFFAOYSA-N | CHEMBL530636 | ZINC000000246761 | 1321660 | MCULE-1723642500 | MolPort-001-958-995
COc1ccc(cc1)c2csc(Nc3cc(C)ccn3)n2 | AT_BP_009 | LOFZQPZBTLGLOP-UHFFFAOYSA-N | CHEMBL4096214 | ZINC000004117008 | 1321662 | MCULE-4324912730 | MolPort-000-690-731
Fc1ccccc1Nc2nc(cs2)c3ccccn3 | AT_BP_010 | VVSAPVJXTYIRNE-UHFFFAOYSA-N | CHEMBL2441361 | ZINC000000813402 | 1509050 | MCULE-1162352817 | MolPort-002-013-555
C(Nc1nc(cs1)c2ccccn2)c3ccccn3 | AT_BP_011 | LLANOFYTSFTNJV-UHFFFAOYSA-N |  |  |  |  | 
O=C(Nc1nc(cs1)c2ccccn2)c3ccccc3 | AT_BP_012 | NNBHDRDMPNNEKO-UHFFFAOYSA-N | CHEMBL164087 |  |  |  | MolPort-004-132-922

**Comment by drc007** [comment]
Hi @bendndi @kym834 @mattodd Given that aminothiazoles are so common perhaps we should look to "crowd source" compounds. Put out a call for people/vendors to search their catalogues/ compound collections and donate a couple of compounds each?

**Comment by bendndi** [comment]
@drc007  That could be an idea. When we first started this project we (DNDi) thoughtwe'd be able to send a set of close analogues, but turned out we no longer had access to them. If there is a way to put put out a crouwdsourcing request we should do so.

**Comment by MFernflower** [comment]
@wwjvdsande @bendndi A common SHDI is commercially available from Sigma NL for screening use (But is a tad pricy) - https://www.sigmaaldrich.com/catalog/product/sial/45371?lang=en&region=NL 

If I had to put lunch money on the target of the aminothiazoles - I'd say they are either enolase or succinate dehydrogenase inhibitors

**Comment by kym834** [comment]
@drc007 @bendndi Agreed. If we can get our hands on some compounds through crowd sourcing we should absolutely do that. I'm happy to still synthesis a fresh sample of the MMV006357 and I can also look into the synthesis of any amionthiazoles that we would like for the SAR that we can't get our hands on either through a crowd source or by purchasing directly.

@bendndi Those compound suggestions look like a good starting point and align with what I was going to suggest, exploring the position of the nitrogen in the pyridine rings and some ring substitutions. I also think that some 4,5-substituted aminothiazoles would be worth looking at. From what I have read this will help with the microsome stability. And those that had lower rates of metabolism (<25% metabolised in 30 minutes) in both hamster and mouse microsome in the data you shared (#2) were all 4,5-substituted aminothiazoles. May not be overly important right now but I think it is still worth seeing what happens when the hydrogen is replaced in terms of activity.

**Comment by MFernflower** [comment]
@drc007 @bendndi @kym834 

Here's what I thought up:

![mandrakeaminothiazole](https://user-images.githubusercontent.com/3164942/93152031-9a8dce00-f6cb-11ea-99ce-fff34f31f9b7.png)

**Comment by wwjvdsande** [comment]
@drc007 @bendndi @kym834
We have tested a succinate dehydrogenase in the past. I don't  We did not publish this yet. It was siccanin 
![image](https://user-images.githubusercontent.com/36398138/95169299-f3282680-07b2-11eb-887a-f021b625c3c6.png). For strain mm55 we had an IC50 of 0.092 ug/ml. The MIC was 4 ug/ml. MICs for 7 other strains ranged were also 4 or 8 ug/ml. The MIC50 was 4 ug/ml. The IC50 measured for M. mycetomatis was comparable to the IC50 of Trichophyton mentagrophytes (IC50 of 0.1 ug/ml) (Nose et al., J Bact, 1971, p 176-184).  Flutolanil is also tested. It was part of the pandemic box. If did not inhibit _M.mycetomatis_ growth at 25 uM and was therefore not tested further.

**Comment by MFernflower** [comment]
@bendndi @mattodd It is also important to note that 2-amino-4-(2-pyridyl)thiazole had human cell toxicity - so fibroblast tox assays are critical for this series!

**Comment by kym834** [comment]
### An update on synthesis

I've mapped out a route to the hit compound (see below). We should be able to access a very wide range of analogue using this route as well.
<img width="500" alt="MMV006357 Synthesis Route" src="https://user-images.githubusercontent.com/61669512/108773854-0f3a3200-75b3-11eb-8641-0fe9a1b9a74c.png">

I'm still waiting on the 2-amino-4-methylpyridine to arrive but have successfully made the brominated intermediate required for the final step of the synthesis along with the 3-pyridinyl, 4-pyridinyl and phenyl analogues. For interest lab notes are [here](https://github.com/TheBreakingGoodProject/ELN-Kymberley-Scroggie)

While I'm waiting, we have 2-aminopyridine in the lab so I'm going to start with the to synthesis the following compounds
<img width="350" atl="Ring-A_analogues" src="https://user-images.githubusercontent.com/61669512/108775193-dac77580-75b4-11eb-8ec9-824ce7e6ae12.png">

If this initial synthesis goes well we are planning to run some of this synthesis route with the SSP students at the University of Sydney later in the semester (April-May). I'll update a little on this in the meeting tomorrow and again once we have work out all the wrinkles.

**Comment by PaulMcKeatingSevenoaks** [comment]
Hi
We’ve been looking at this at Sevenoaks School as a possible next place to look after some fenarimols. It’d be interesting to find out how it’s going for you and if we could contribute in the future.

**Comment by kym834** [comment]
Hi All, 

MMV0006357 has been synthesised and characterised and is ready to be shipped for testing (after it has been assigned a MYOS number (#5)).

Here is the synthesis route that I used

<img width="500" src="https://user-images.githubusercontent.com/61669512/121986353-ad0f7200-cdd9-11eb-9d04-39e298c1bcdd.png">

I've found it to be a robust synthesis route and we have used it to make another 12 analogues with first year students at The University of Sydney. I'm going to write up a summary of what we did in more detail in a new issue

@PaulMcKeatingSevenoaks This went quite well for us and I think there is a lot of room to build this series with high school and undergraduate students.

**Comment by wwjvdsande** [comment]
That's great. When ready please let us know.

**Comment by PaulMcKeatingSevenoaks** [comment]
This looks really promising, thanks, @kym834

(And a cool ring-forming reaction for school students!)
